docs/source/optimization_ov.mdx

```diff
@@ -112,18 +118,98 @@ metrics = trainer.evaluate()
+
+```diff
+-from optimum.intel.openvino.trainer import OVConfig, OVTrainer
++from optimum.intel.openvino import OVConfig, OVTrainer, OVTrainingArguments
+
+# Load teacher model
++teacher_model = AutoModelForSequenceClassification.from_pretrained(teacher_model_or_path)
+
+-ov_config = OVConfig()
++ov_config = OVConfig(compression=compression_config)
+
+trainer = OVTrainer(
+    model=model,
++   teacher_model=teacher_model,
+-    args=TrainingArguments(save_dir, num_train_epochs=1.0, do_train=True, do_eval=True),
++    args=OVTrainingArguments(save_dir, num_train_epochs=1.0, do_train=True, do_eval=True, distillation_temperature=3, distillation_weight=0.9),
+    train_dataset=dataset["train"].select(range(300)),
+    eval_dataset=dataset["validation"],
+    compute_metrics=compute_metrics,
+    tokenizer=tokenizer,
+    data_collator=default_data_collator,
++   ov_config=ov_config,
+    task="sequence-classification",
+)
+
+# Train the model like usual, internally the training is applied with pruning, quantization and distillation
+train_result = trainer.train()
+metrics = trainer.evaluate()
+# Export the quantized model to OpenVINO IR format and save it
+trainer.save_model()
+```
+
+More on the description and how to configure movement sparsity, see NNCF documentation [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/torch/sparsity/movement/MovementSparsity.md).
+
+More on available algorithms in NNCF, see documentation [here](https://github.com/openvinotoolkit/nncf/tree/develop/docs/compression_algorithms).
+
+For complete JPQD scripts, please refer to examples provided [here](https://github.com/huggingface/optimum-intel/tree/main/examples/openvino). 
+
+
 ## Inference with Transformers pipeline
 
-After applying quantization on our model, we can then easily load it with our [`OVModelForXxx`](https://huggingface.co/docs/optimum/intel_inference#optimum-inference-with-openvino) classes and perform inference with OpenVINO Runtime using the Transformers [pipelines](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines).
+After applying quantization on our model, we can then easily load it with our `OVModelFor<Task>` classes and perform inference with OpenVINO Runtime using the Transformers [pipelines](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines).
 
 ```python
 from transformers import pipeline
 from optimum.intel.openvino import OVModelForSequenceClassification
 
-ov_model = OVModelForSequenceClassification.from_pretrained(save_dir)
+ov_model = OVModelForSequenceClassification.from_pretrained(model_dir)
```

Comment:
I would keep `save_dir` for users to load the quantized model obtained through by either PTQ or QAT, or use a valid model id such as  `"helenai/distilbert-base-uncased-finetuned-sst-2-english-ov-int8"`, as currently `model_dir` is not defined anywhere

Reply:
@echarlaix Thanks for the suggestion, revised per your suggestion [16da776](https://github.com/huggingface/optimum-intel/pull/150/commits/16da7763b1a725f0389165395ca7cdd5e0b207d8)